core/arch/arm/plat-mediatek/main.c

```diff
@@ -38,6 +38,14 @@
 
 static void main_fiq(void);
 
+register_phys_mem(MEM_AREA_IO_NSEC,
+		  ROUNDDOWN(CONSOLE_UART_BASE, CORE_MMU_DEVICE_SIZE),
```

Comment:
The `ROUNDDOWN()` isn't needed, that's taken care of when the address ranges are collected.